action/GlobalPlanner.action

```diff
@@ -1,5 +1,9 @@
+# This is a modified version of GlobalPlanner.action from ros-planning/moveit_msgs.
+# A JointTrajectory was added so that timing information is retained.
+
 # Motion planning request to the global planner
 string planning_group
+trajectory_msgs/JointTrajectory trajectory
```

Comment:
I don't really follow why this should be needed, but why not add a reference trajectory here? https://github.com/ros-planning/moveit_msgs/blob/master/msg/MotionPlanRequest.msg#L25

Reply:
So, the moveit_msgs/MotionSequenceRequest contains a JointTrajectory if you dig deep enough.

MotionSequenceRequest-> MotionSequenceItem[] -> MotionPlanRequest-> GenericTrajectory[]-> JointTrajectory[]

I guess this new field is probably not needed :(

What do you think @henningkayser? Is it too hard to dig all the way down to JointTrajectory in the current message type?

Reply:
I'm cool with either approach. I guess using what currently exists (even if you do need to dig down somewhat deep) makes the most sense so I can make that change

Reply:
I'm reconsidering this. I think having a dedicated "trajectory" field is more intuitive to the user to assign than having to dig  deep in the motionplanrequest. This is for three reasons:

1. The way the HP will check to see if it should process the already planned trajectory (as opposed to planning one) is by specifically looking in this "trajectory" field to see if has a multi-point trajectory. This makes this field pretty important to this action. Having it buried down within the MotionSequenceRequest could therefore be less intuitive to the user.
2. Having this trajectory field outside the MotionSequenceRequest highlights that this action isn't so much a request to plan as it is to just bypass planning and execute via the local planner.
3. The action already conains a field for the `planning_group` so wouldn't be a big stretch to add the accompanying trajectory on the same level if applicable.

All this said, if you really think using the reference-trajectory field is best, I can run with that and make the necessary changes.